deadlines/src/main/java/com/palantir/deadlines/DeadlineExpiredException.java

```diff
@@ -31,6 +31,8 @@ private DeadlineExpiredException(String message) {
         super(message);
     }
 
+    public abstract int httpStatusCode();
```

Comment:
Slight preference to put these into a helper method on `Deadlines.java` along the lines of `int getHttpStatusCode(DeadlineExpiredException)`, that way the exceptions themselves aren't directly coupled to http.

Reply:
works for me, will fix